src/palindrom.js

```diff
@@ -163,9 +167,16 @@ const Palindrom = (() => {
       if (scheduledError) {
         return;
       }
-      scheduledError = setTimeout(() => {
+      var scheduledError = setTimeout(() => {
         scheduledError = null;
-        onError(); // timeout has passed and response hasn't arrived
+        onError(
+          new PalindromConnectionError(
+            "Timeout has passed and response hasn't arrived",
+            'Client',
```

Comment:
Can this be a `const`?

Reply:
I don't get it. Could you elaborate please?

Reply:
like `const CLIENT = 'Client'; in line no 1, and just